proto/connect/trust_zone_service/v1alpha1/trust_zone_service.proto

```diff
@@ -78,3 +84,12 @@ message RegisterAgentRequest {
 message RegisterAgentResponse {
   string agent_id = 3;
 }
+
+message RegisterTrustZoneServerRequest {
+  TrustZoneServer trust_zone_server = 1;
+  spire.api.types.Bundle bundle = 2;
+}
+
+message RegisterTrustZoneServerResponse {
+  bool success = 1;
```

Comment:
We moved away from success in the response to gRPC status for error reporting.

Reply:
Is there any info we want to respond with here? Or just an empty response?